src/SwedbankPay.Sdk.Infrastructure/PaymentOrders/PaymentOrdersResource.cs

```diff
@@ -32,23 +22,32 @@ public PaymentOrdersResource(HttpClient httpClient) : base(httpClient)
         /// <exception cref="InvalidOperationException"></exception>
         /// <exception cref="HttpRequestException"></exception>
         /// <exception cref="HttpResponseException"></exception>
-        public Task<PaymentOrder> Get(Uri id, PaymentOrderExpand paymentOrderExpand = PaymentOrderExpand.None)
+        public async Task<IPaymentOrderResponse> Get(Uri id, PaymentOrderExpand paymentOrderExpand = PaymentOrderExpand.None)
         {
             if (id == null)
                 throw new ArgumentNullException(nameof(id), $"{id} cannot be null");
 
-            return GetInternalAsync(id, paymentOrderExpand);
+            Uri url = id.GetUrlWithQueryString(paymentOrderExpand);
+
+            var paymentOrderResponseContainer = await HttpClient.GetAsJsonAsync<PaymentOrderResponseDto>(url);
+
+            return new PaymentOrderResponse(paymentOrderResponseContainer, HttpClient);
         }
 
         /// <summary>
-        ///     Get payment order by id
+        ///     Create a payment order
         /// </summary>
-        /// <param name="id"></param>
+        /// <param name="paymentOrderRequest"></param>
         /// <param name="paymentOrderExpand"></param>
         /// <returns></returns>
-        private async Task<PaymentOrder> GetInternalAsync(Uri id, PaymentOrderExpand paymentOrderExpand = PaymentOrderExpand.None)
+        public async Task<IPaymentOrderResponse> Create(PaymentOrderRequest paymentOrderRequest,
+                                               PaymentOrderExpand paymentOrderExpand = PaymentOrderExpand.None)
```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)